source/includes/_install_js_spa_intro.md

```diff
@@ -25,57 +25,167 @@
 * When you re-load the page, you should see Wootric survey show up.
 * You are done.
 
-## Angular Step 1: Create Factory
+## AngularJS Example (using Factory)
 
 ```javascript
-<!-- begin Wootric code -->
-angular.module('wootricAngularApp')
-.factory('wootricFactory', function () {
-window.wootricSettings = {
-    email: 'nps@example.com',// TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
+// 1. Load Wootric JS SDK
+<script type="text/javascript" src="https://cdn.wootric.com/wootric-sdk.js"></script>
+
+// 2. Create Factory
+app.factory('wootricFactory', function() {

```

Comment:
Where does the `app` var come from? I thought using `angular` was more general.

Reply:
@diegoserranoa thanks for checking. These are code snippets and assumes that a developer looking at docs knows at least very basic also in point number for I have mentioned that there is code example for this on github and when `README` will be checked out & thus the code itself then there's `app` variable  https://github.com/Wootric/spa-examples/blob/master/angularjs/app/app.js#L4